PullRequestCreated/ReadMe.md

```diff
@@ -3,7 +3,9 @@
 Upon pull request creation, create a Jira ticket if no tickets are mentioned in the title.
 
 Depending on the content of the PR description, this action will create different types of Jira tickets:
-- A `Task` with no parent if there is no ticket mentioned in the description
+- A `Task` with the mentioned `Epic` as a parent
+- A `Sub-task` with the first ticket mentioned as a parent, except `Sub-task`
```

Comment:
The docs is currently out of date, as the "first" will be done in #106. It can probably stay that way for the moment